boards/arm/nrf7002dk_nrf5340/nrf7002dk_nrf5340_cpuapp.dts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf5340_cpuapp_qkaa.dtsi>
+#include "nrf5340_cpuapp_common.dts"
+
+/ {
+	model = "Nordic NRF7002 DK NRF5340 Application";
+	compatible = "nordic,nrf7002-dk-nrf5340-cpuapp";
+
+	chosen {
+		zephyr,sram = &sram0_image;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+		zephyr,sram-secure-partition = &sram0_s;
+		zephyr,sram-non-secure-partition = &sram0_ns;
+	};
+};
+
+&spi2 {
+	compatible = "nordic,nrf-spim";
+	status = "okay";
+	sck-pin = <47>;
+	miso-pin = <46>;
+	mosi-pin = <45>;
+};
```

Comment:
This creates a conflict between `spi2` and `spi4` (both are assigned with the same pins), so it should be removed.